ref-exchange/src/multi_fungible_token.rs

```diff
@@ -32,8 +32,24 @@ enum TokenOrPool {
     Pool(u64),
 }
 
+/// [AUDIT_06]
+/// This is used to parse token_id fields in mft protocol used in ref,
+/// So, if we choose #nn as a partern, should announce it in mft protocol.
+/// cause # is not allowed in a normal account id, it can be a partern leading char
```

Comment:
I'm not sure where NFT standard end up, but one option is to use the ":" symbol - it was similar idea that it's no allowed in the accounts and it allows to create "deep" link into something inside a contract: "contract_name:token_id". Not sure it's useful to define this here and using something is better than nothing.

Reply:
changed from `#` to `:`